scripts/Packages/Package_BGA/bga.yaml

```diff
@@ -1,114 +1,140 @@
+  body_size_x: 3.415
+  body_size_y: 3.535
+  overall_height: 0.625
   pitch: 0.4
   pad_diameter: 0.225
   layout_x: 8
   layout_y: 8
   mask_margin: 0.05
 Texas_DSBGA-8_1.5195x1.5195mm_Layout3x3_P0.5mm:
-  description: "Texas Instruments, DSBGA, 1.5195x1.5195x0.600mm, 8 ball 3x3 area grid, YZR pad definition (http://www.ti.com/lit/ml/mxbg270/mxbg270.pdf)"
-  pkg_width: 1.5195
-  pkg_height: 1.5195
+  description: "Texas Instruments, DSBGA, area grid, YZR pad definition"
+  size_source: "http://www.ti.com/lit/ml/mxbg270/mxbg270.pdf"
+  body_size_x: 1.5195
+  body_size_y: 1.5195
+  overall_height: 0.6
   pitch: 0.5
   pad_diameter: 0.2625
   layout_x: 3
   layout_y: 3
   mask_margin: 0.05
   row_skips: [[], [2], []]
 Texas_MicroStar_Junior_BGA-113_7.0x7.0mm_Layout12x12_P0.5mm:
-  description: "Texas Instruments, BGA Microstar Junior, 7x7mm, 113 ball 12x12 grid, NSMD pad definition (http://www.ti.com/lit/ml/mpbg674/mpbg674.pdf, http://www.ti.com/lit/wp/ssyz015b/ssyz015b.pdf)"
-  pkg_width: 7
-  pkg_height: 7
+  description: "Texas Instruments, BGA Microstar Junior"
+  size_source: "http://www.ti.com/lit/ml/mpbg674/mpbg674.pdf, http://www.ti.com/lit/wp/ssyz015b/ssyz015b.pdf"
+  body_size_x: 7
+  body_size_y: 7
   pitch: 0.5
   pad_diameter: 0.25
   layout_x: 12
   layout_y: 12
   mask_margin: 0.025
   row_skips: [[], [], [[4, 11]], [3, 10], [3, 10], [3, 6, 7, 10], [3, 6, 7, 10], [3, 10], [3, 10], [[3, 11]], [], []]
 TFBGA-64_5x5mm_Layout8x8_P0.5mm:
-  description: "TFBGA-64, 8x8 raster, 5x5mm package, pitch 0.5mm; http://www.st.com/resource/en/datasheet/stm32f100v8.pdf#page=83"
-  pkg_width: 5.0
-  pkg_height: 5.0
+  description: "TFBGA-64"
+  size_source: "http://www.st.com/resource/en/datasheet/stm32f100v8.pdf#page=83"
+  body_size_x: 5.0
+  body_size_y: 5.0
   pitch: 0.5
   pad_diameter: 0.28
   mask_margin: 0.045
   paste_margin: 0.000001
   layout_x: 8
   layout_y: 8
 TFBGA-100_8x8mm_Layout10x10_P0.8mm:
-  description: "TFBGA-100, 10x10 raster, 8x8mm package, pitch 0.8mm; http://www.st.com/resource/en/datasheet/stm32f746zg.pdf#page=199"
-  pkg_width: 8.0
-  pkg_height: 8.0
+  description: "TFBGA-100"
+  size_source: "http://www.st.com/resource/en/datasheet/stm32f746zg.pdf#page=199"
+  body_size_x: 8.0
+  body_size_y: 8.0
   pitch: 0.8
   pad_diameter: 0.4
   mask_margin: 0.035
   paste_margin: 0.000001
   layout_x: 10
   layout_y: 10
 TFBGA-121_10x10mm_Layout11x11_P0.8mm:
-  description: "TFBGA-121, 11x11 raster, 10x10mm package, pitch 0.8mm; http://ww1.microchip.com/downloads/en/PackagingSpec/00000049BQ.pdf#p495"
-  pkg_width: 10.0
-  pkg_height: 10.0
+  description: "TFBGA-121"
+  size_source: "http://ww1.microchip.com/downloads/en/PackagingSpec/00000049BQ.pdf#p495"
+  body_size_x: 10.0
+  body_size_y: 10.0
   pitch: 0.8
   pad_diameter: 0.4
   mask_margin: 0.035
   paste_margin: 0.000001
   layout_x: 11
   layout_y: 11
+TFBGA-196_11.0x11.0mm_Layout14x14_P0.75mm_Ball0.32mm_Pad0.35mm_SMD:
```

Comment:
The ball diameter does not need to be specified, I don't think. What was the reason to add it?